config/routes.rb

```diff
@@ -1,10 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
-
-  # Defines the root path route ("/")
-  # root "posts#index"
+  root "games#index"
```

Comment:
There is also an option to use `resource :games`, but having a root definition to default path is standard. One handy reference is to checkout `http://localhost:3000/rails/info/routes` which can provide you the path helpers.

Reply:
I have incorporated resource :games in a later branch.